### ISSUE TITLE
Foreign_key test report additional/missing values

### ISSUE THREAD

**Issue by semcogli** [urbansim/orca_test#11]
Foreign key test reports errors but not the values causing those errors. It would be much helpful to print out those mismatching values as well. Thanks.

**Comment by smmaurer** [comment]
Yes, agreed. This will take a little bit of work -- adding logic to the relevant assertion functions, and figuring out the best way to attach this kind of info to the OrcaAssertionError object. Leaving this open as a feature request.
